src/components/RightPane/SectionTable/PrereqTree.tsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable prefer-const */
+import { FC } from 'react';
+import { Grid, Popup } from 'semantic-ui-react';
+
+import { PrerequisiteJSONNode, PrerequisiteJSON } from '../../../lib/peterportal.types';
```

Comment:
Try using `$lib` here instead of the relative path. See the `import {isDarkMode...` line as an example.